### PR TITLE
docs(v2): Rewrite markdown images section

### PR DESCRIPTION
**Pull request by ramiy**
Before it looked like we have two ways to display images, now it's three ways. Each syntax has a separate example.

This way it's clear, I see each method and the code example.

Before there were 2 methods in a single code block.

### CODE REVIEW

File: website/docs/guides/markdown-features/markdown-features-assets.mdx [hunk 1]
```diff
@@ -22,29 +22,32 @@ Let's imagine the following file structure:
 
 ## Images {#images}
 
-You can use images in Markdown, or by requiring them and using a JSX image tag:
+You can display images in three diffirent ways: Markdown syntax, JSX require or ES imports syntax.
```
Comment (slorber) [inline website/docs/guides/markdown-features/markdown-features-assets.mdx:25]:
```suggestion
You can display images in three different ways: Markdown syntax, JSX require or ES imports syntax.
```

File: website/docs/guides/markdown-features/markdown-features-assets.mdx [hunk 2]
```diff
@@ -22,29 +22,32 @@ Let's imagine the following file structure:
...
+
+Display images using simple Markdown syntax:
 
 ```mdx
-# My Markdown page
+![Example banner](./assets/docusaurus-asset-example-banner.png)
+```
 
+Require images using JSX image tag:
```
Comment (slorber) [inline website/docs/guides/markdown-features/markdown-features-assets.mdx:33]:
```suggestion
Display images using inline CommonJS `require` in JSX image tag:
```

File: website/docs/guides/markdown-features/markdown-features-assets.mdx [hunk 3]
```diff
@@ -22,29 +22,32 @@ Let's imagine the following file structure:
...
   alt="Example banner"
 />
-
-or
-
-![Example banner](./assets/docusaurus-asset-example-banner.png)
 ```
 
-The ES imports syntax also works:
+Show images using ES imports syntax:
```
Comment (slorber) [inline website/docs/guides/markdown-features/markdown-features-assets.mdx:42]:
```suggestion
Display images using ES `import` syntax and JSX image tag:
```

**Review by slorber — APPROVED** [review]
Suggesting some text changes for consistency

**Comment by ramiy** [comment]
@slorber thanks you for the text changes!

**Pull request merged**